pkg/crc/constants/constants.go

```diff
@@ -25,8 +26,27 @@ const (
 	GlobalStateFile      = "globalstate.json"
 	CrcLandingPageURL    = "https://cloud.redhat.com/openshift/install/crc/installer-provisioned" // #nosec G101
 	PullSecretFile       = "pullsecret.json"
+
+	DefaultOcURLDarwin  = "https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/macosx/oc.tar.gz"
+	DefaultOcURLLinux   = "https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz"
+	DefaultOcURLWindows = "https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/windows/oc.zip"
```

Comment:
Why we need those `DefaultOcURL<OS>` variables when we are getting it from `GetOcUrl` side?

Reply:
I forgot to remove these :) Thanks!